### ISSUE TITLE
Defaults in /etc/fonts/conf.d/10-*

### ISSUE THREAD

**Issue by vaskark** [NerdyPepper/scientifica#38]
I'd like to know what files I would need to restore in `/etc/fonts/conf.d/` if I decide to reverse the instructions provided for this font.
Thanks. I'm on Ubuntu 19.10.

**Comment by NerdyPepper** [comment]
Good point, if you haven't already deleted those files, I would suggest moving them to the `conf.avail` directory:

```shell
sudo mv ./10* ../conf.avail/ 
sudo mv ./70-no-bitmaps.conf ../conf.avail
```

If you have already deleted them, these might come in handy:

 - [10-hinting-slight.conf](https://u.peppe.rs/B0.conf)
 - [10-scale-bitmap-fonts.conf](https://u.peppe.rs/lN.conf)
 - [10-sub-pixel-rgb.conf](https://u.peppe.rs/7d.conf)
 - [70-no-bitmap-fonts.conf](https://u.peppe.rs/Te.conf)

I am currently working on rewriting the readme from scratch, please bear with me until the changes are made :)

**Comment by vaskark** [comment]
Thanks for responding.
My further searches resulted in the exact same advice. Good to know how to undo things ;)

It's a brilliant-looking font. I'm really enjoying it. Especially the ttf version.
I did notice something, though.

Two of your powerline extras symbols seem to be out of place:

`E0B5` should be `E0D2`
`E0B4` should be `E0D4`

Once again, many thanks.

**Comment by NerdyPepper** [comment]
>E0B5 should be E0D2
E0B4 should be E0D4

thanks for pointing those out! they have been fixed in the 2.0 release, along with a bunch of other powerline glyph additions!
